### DISCUSSION TITLE
[Script request] Zammad Ticketsystem

### DISCUSSION THREAD

**Discussion by florianwilde** [community-scripts/ProxmoxVE#108] [Request script]
### Application Name

Zammad

### Website

https://docs.zammad.org/en/latest/install/package.html

### Description

Zammad is a web based open source helpdesk/customer support system with many features to manage customer communication via several channels like telephone, facebook, twitter, chat and emails. It is distributed under version 3 of the GNU AFFERO General Public License (GNU AGPLv3).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by second2falcon** [comment]
Take my upvote. I tried running Zammad in an LXC a while ago, but didn't get it to work. VM works pretty well.

**Comment by MickLesk** [comment]
Added.
